state/rethinkdb/rethinkdb.go

```diff
@@ -0,0 +1,364 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package rethinkdb
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"time"
+
+	r "github.com/dancannon/gorethink"
+	"github.com/dapr/components-contrib/state"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/pkg/errors"
+)
+
+const (
+	stateTableName          = "daprstate"
```

Comment:
looks like table name is fixed. as the initial pr, it is fine. maybe it would be good to allow users to use custom table name via metadata in CRD

Reply:
added parameterized table option